bindata/network/sriov/sriov-device-plugin.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: sriov-device-plugin
+  namespace: sriov-device-plugin
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the SR-IOV Network Device Plugin.
+spec:
+  selector:
+    matchLabels:
+      app: sriovdp
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: sriovdp
+        component: network
+        type: infra
+        openshift.io/component: network
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      tolerations:
+      - operator: Exists
+      serviceAccountName: sriov-device-plugin
+      containers:
+      - name: sriovdp
+        image: {{.SRIOVDevicePluginImage}}
+        args:
+        - --log-dir=sriovdp
```

Comment:
please replace the args with "--log-level=10"

```
args:
- --log-level=10
```

--log-dir will cause failure when creating the log file since there is no log dir volume mounted in this daemonset.